### ISSUE TITLE
Missing documentation on how to update existing devices

### ISSUE THREAD

**Issue by msavazzi** [alandtse/alexa_media_player#2511]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/alandtse/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

I have added other Alexa and changed name of existing ones but they are not reflected in HA

**To Reproduce**

1) install 1 alexa
2) configure the integration
3) add a new alexa and rename the first one
4) check integration -> 

**Expected behavior**

devices in HA to be updated and added 

**Screenshots**

no screenshot

**System details**

- Home Assistant version: Core 2024.9.1
- alexa_media version (from `const.py` or HA startup log): 4.12.12
- alexapy version (from `pip show alexapy` in homeasssistant container or HA startup log):
- Is Amazon [2FA/2SV enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account)  (y/n): Y
- Amazon Domain: .it

**Debug Logs (alexa_media & alexapy)**
Please provide [logs](https://github.com/alandtse/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).
[home-assistant_alexa_media_2024-09-07T16-42-42.190Z.log](https://github.com/user-attachments/files/16919199/home-assistant_alexa_media_2024-09-07T16-42-42.190Z.log)

**Additional context**

**Comment by danielbrunt57** [comment]
Did you reload the integration after step 3?

![image](https://github.com/user-attachments/assets/fae66124-52f8-47b0-9ce5-3828b338a910)

**Comment by msavazzi** [comment]
Yes I did yesterday but did not changed...
I did it today and it worked :)

heheheh weird.

<img width="841" alt="image" src="https://github.com/user-attachments/assets/8e52cb37-7488-4d38-b968-357aed5c87fa">
